README.md

```diff
@@ -3,5 +3,5 @@
 <a href="https://github.com/me-pic">
    <img src="https://avatars.githubusercontent.com/u/77584086?v=4?s=100" width="100px;" alt=""/>
    <br /><sub><b>Marie-Eve Picard</b></sub>
-   <br /> Master's student in Psychology at Université de Montréal</b> 	
+   <br /> I am a super cool Master's student in Psychology at Université de Montréal</b> 	
```

Comment:
I suggest inserting this information as text above your picture. The html formatting is a bit akward here because the text is long.